dev/docker/doris/Dockerfile

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright 2024 Datastrato Pvt Ltd.
+# This software is licensed under the Apache License version 2.
+#
+FROM ubuntu:22.04
+LABEL maintainer="support@datastrato.com"
+
+ARG DORIS_PACKAGE_NAME
+ARG DORIS_FILE_NAME
+ARG TARGET_ARCH
+
+WORKDIR /
+
+################################################################################
+# update and install basic tools
+RUN apt-get update -y &&  DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    patchelf gdb binutils binutils-common mysql-client \
+    curl wget less vim htop iproute2 numactl jq iotop sysstat tzdata xz-utils \
+    tcpdump iputils-ping dnsutils strace lsof blktrace \
+    bpfcc-tools linux-headers-realtime linux-tools-realtime silversearcher-ag \
+    net-tools openjdk-8-jdk && \
+    rm -rf /var/lib/apt/lists/*
+
+################################################################################
+# set environment variables
+ENV JAVA_HOME=/usr/local/jdk
+ENV DORIS_HOME=/opt/apache-doris
+ENV DORIS_FE_HOME=${DORIS_HOME}/fe
+ENV DORIS_BE_HOME=${DORIS_HOME}/be
+
+ENV PATH=${JAVA_HOME}/bin:${DORIS_FE_HOME}/bin:${DORIS_BE_HOME}/bin:${PATH}
+
+################################################################################
+# setup java
+
+RUN ARCH=$(uname -m) && \
+  if [ "$ARCH" = "aarch64" ] || [ "$ARCH" = "arm64" ]; then \
+    ln -s /usr/lib/jvm/java-8-openjdk-arm64 ${JAVA_HOME}; \
+  else \
+    ln -s /usr/lib/jvm/java-8-openjdk-amd64 ${JAVA_HOME}; \
+  fi
+
+#################################################################################
+## add files
+
+ADD packages/${DORIS_FILE_NAME} /opt
+RUN ln -s /opt/${DORIS_PACKAGE_NAME} ${DORIS_HOME}
+
+COPY start.sh ${DORIS_HOME}
+COPY start_be.sh ${DORIS_BE_HOME}/bin
+
+################################################################################
+# set WORKDIR/EXPOSE/CMD
+
+WORKDIR ${DORIS_HOME}
+
+EXPOSE 8030
+EXPOSE 8040
+EXPOSE 9030
+
+CMD ["bash", "/opt/apache-doris/start.sh"]
```

Comment:
we should not use ${DORIS_HOME} here.
```
bash: ${DORIS_HOME}/start.sh: No such file or directory
```
seems `DORIS_HOME` just worked when docker build.

Reply:
If so, I don't think you need the environment variant `DORIS_HOME` in the build file.

Reply:
it used in L47 & L49.
do you mean replace it to another variant ?

Reply:
I mean, since we can't guarantee the value of `DORIS_HOME` and `/opt/apache-doris` are equal if you use constant value `"/opt/apache-doris/start.sh"` in the entry point of the image, I think we'd better think how to more them uniform.